lib/logdna/client.rb

```diff
@@ -50,7 +50,6 @@ def schedule_flush
         flush if @flush_scheduled
       }
       thread = Thread.new { start_timer.call }
```

Comment:
I meant this variable. thread . We won't be able to merge the diff with the unused variable

Reply:
👍